addOns/websocket/src/main/zapHomeFiles/scripts/templates/websocketpassive/Email Disclosure.js

```diff
@@ -0,0 +1,33 @@
+// This script scans incoming WebSocket messages for email address.
+
+// * Author: Manos Kirtas (Manos Kirtas)
+// * Based on: community-scripts/passive/Find Emails.js
+
+OPCODE_TEXT = 0x1;
+RISK_INFO = 0;
+CONFIDENCE_HIGH = 3;
+
+var emailRegex = new RegExp("([a-zA-Z0-9_.+-]+@[a-zA-Z0-9]+[a-zA-Z0-9-]*\.[a-zA-Z0-9-.]*[a-zA-Z0-9]{2,})", 'gmi');
```

Comment:
If it's flagged `gmi` (`i` specifically) do you need to specify both `a-zA-Z`?

Reply:
Fixed in 450043c95